src/main/scala/mesosphere/marathon/api/v2/TasksResource.scala

```diff
@@ -106,6 +108,7 @@ class TasksResource @Inject() (
   @Path("delete")
   def killTasks(
     @QueryParam("scale")@DefaultValue("false") scale: Boolean,
+    @QueryParam("force")@DefaultValue("false") force: Boolean,
```

Comment:
Can you add tests for `force = true`? This param is
- only evaluated for `scale = true`
- not documented at all
Fixing the second point could clarify the first.